bin/run_setup.sh

```diff
@@ -21,6 +21,11 @@ python ./bin/s3_cli.py delete "${CRASHMOVER_CRASHSTORAGE_BUCKET_NAME}"
 python ./bin/s3_cli.py create "${CRASHMOVER_CRASHSTORAGE_BUCKET_NAME}"
 python ./bin/s3_cli.py list_buckets
 
+echo "Delete and create GCS bucket..."
+python ./bin/gcs_cli.py delete "${CRASHMOVER_CRASHSTORAGE_BUCKET_NAME}"
```

Comment:
This makes me feel a little uneasy. If it were me, I'd keep a separate set of configuration for GCS than S3 and not mix them. Ditto for PubSub and SQS. 

It makes it a little easier to answer these questions:

1. What are all the things that need to be configured for GCS? This is helpful for configuring these things in stage/prod environments.
2. What are all the configuration we're no longer using? This is helpful when we're cleaning everything up after we've migrated to GCP.

Ergo, I kind of think we should be using separate configuration. I'd probably name the new ones something like `CRASHMOVER_CRASHSTORAGE_GCS_*` and `CRASHMOVER_CRASHPUBLISH_PUBSUB_*`.

Reply:
If it were me, I'd do it because it's easier to know what to remove from `configmap.yaml`. But I'm game for whatever you're comfortable with for both GCS and PubSub config here.